### PR TITLE
chore: update eslint and apply formatting changes

### PR DESCRIPTION
**Pull request by helendurranttw**
Update Eslint and associated dependencies

## 📎 Jira ticket

[DS-373](https://transferwise.atlassian.net/browse/DS-373)

## ❓ Context 

Part of a wider dependency update.

## 🚀 Changes

* Updates Eslint and associated deps
* Turns off one new rule that caused hundreds of errors
* Applied formatting changes

## ✅ Checklist

- [ ] All changes are covered by tests
- [ ] The changes are covered in docs
- [ ] The commits follow conventional commits standards

### CODE REVIEW

File: packages/docs/package.json [hunk 1]
```diff
@@ -53,15 +53,15 @@
     "react-live": "^2.2.2",
     "refractor": "^3.0.0",
     "rehype": "^10.0.0",
-    "rehype-slug": "^3.0.0"
+    "rehype-slug": "^3.0.0",
+    "react": "16.8.6",
```
Comment (andreapiras) [inline packages/docs/package.json:57]: can this be in root ? and also version 16.13.1?

File: packages/docs/package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "@transferwise/icons": "^1.5.0",
     "@transferwise/neptune-css": "^0.13.1",
     "@zeit/next-css": "^1.0.1",
-    "bootstrap": "transferwise/bootstrap",
+    "bootstrap": "github:transferwise/bootstrap#^v5.20.0",
```
Comment (andreapiras) [inline packages/docs/package.json:39]: Should this be in root?

File: packages/docs/.eslintrc [hunk 3]
```diff
@@ -1,10 +1,14 @@
 {
```
Comment (andreapiras) [inline packages/docs/.eslintrc:1]: We need to look into centralise this file too

File: packages/css/package.json [hunk 4]
```diff
@@ -30,8 +30,7 @@
     "docs": "gulp lessWatcher",
     "copy:images": "cpx 'src/img/**' dist/img",
     "copy:fonts": "cpx 'src/fonts/*' dist/fonts",
-    "lint": "npm-run-all lint:*",
-    "lint:prettier": "prettier --find-config-path --write src/less/*.less",
+    "format": "prettier --write src/less/*.less",
```
Comment (andreapiras) [inline packages/css/package.json:33]: should we have this also in root? I'd leave prettier in the naming even ```format:prettier```

Comment (helendurranttw) [reply packages/css/package.json:33]: I'll rename it to prettier - I prefer to leave the : for when there are multiple tasks or sub tasks.

**Pull request merged**